test/integration/helpers_test.go

```diff
@@ -704,3 +704,17 @@ func CopyDir(src, dst string) error {
 
 	return nil
 }
+
+// getKubernetesVersionsForTesting returns unique Kubernetes versions to test against
+func getKubernetesVersionsForTesting() []string {
```

Comment:
The get prefix is not needed. It should be used if this preform a Get request or something similar. KubernetesVersionsForTesting is good but we are in the context of the tests, so we can remove the ForTesting suffix.

Rename to KubernetesVersions()?